packages/pluggableWidgets/image-viewer-web/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "image-viewer-web",
+  "widgetName": "ImageViewer",
+  "version": "2.0.0",
+  "description": "Display an image and enlarge it on click",
+  "copyright": "Mendix BV",
```

Comment:
Maybe we can update following the newest widgets with real copyright

Reply:
Created story for this WC-547